doc/btp_gap.txt

```diff
@@ -656,6 +656,40 @@ Commands and responses:
 
                 In case of an error, the error response will be returned.
 
+        Opcode 0x2a - Pair V2 command/response
+
+                Controller Index:       <controller id>
+                Command parameters:     Address_Type (1 octet)
+                                        Address (6 octets)
+                                        Mode (1 octet)
+                                        Level (1 octet)
+                                        Flags (1 octet)
+                Return Parameters:      <none>
+
+                This command is used to initiate security with remote. If
+                peer is already paired IUT is expected to enable security
+                (encryption) with peer. If peer is not paired IUT shall
+                start pairing process.
```

Comment:
I'd suggest to add  to allowed parameters
0xff = Any Mode
0xff = Any Level


That way calling Pair V2 with those set to 0xff and flags=0 would mimic old Pair command behavior